content/200-concepts/100-components/02-prisma-client/053-client-extensions/130-result.mdx

```diff
@@ -68,22 +70,22 @@ const xprisma = prisma.$extends({
   },
 })
 
-const user = await xprisma.user.findFirst()
+const user = await prisma.user.findFirst()
```

Comment:
If we want to use `extendedPrisma` in this file as well, then this should be changed.
```suggestion
const user = await extendedPrisma.user.findFirst()
```